package.json

```diff
@@ -1,19 +1,19 @@
 {
   "name": "mtgjson-website",
-  "version": "4.7.0",
+  "version": "2.0.0",
```

Comment:
Can you go backwards?

Reply:
Yes because this is not a package - its more like metadata for a project like this and really only is for the developers, not anyone else.